python/setup.py

```diff
@@ -6,27 +6,32 @@
 from setuptools.dist import Distribution
 from platform import system
 
-# Universal build will only contain python files, and libdlr.so is expected to be found in compiled
-# model artifact folder.
-is_universal = "--universal" in sys.argv
+if "--universal" in sys.argv:
+  raise ValueError("Creating py2.py3 wheels is not supported")
 
 CURRENT_DIR = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
 
-if not is_universal:
-    # Use libpath.py to locate libdlr.so
-    LIBPATH_PY = os.path.abspath('./dlr/libpath.py')
-    LIBPATH = {'__file__': LIBPATH_PY}
-    exec(compile(open(LIBPATH_PY, "rb").read(), LIBPATH_PY, 'exec'),
-        LIBPATH, LIBPATH)
-    LIB_PATH = [os.path.relpath(LIBPATH['find_lib_path'](setup=True), CURRENT_DIR)]
+BUILD_DIR = "../build/lib/"
```

Comment:
Looks like it works for Windows as well.
According to "Install DLR on Windows" - "You will see dlr.dll and dlr.lib in the `neo-ai-dlr\build\lib` directory.
https://github.com/neo-ai/neo-ai-dlr/blob/master/sagemaker-neo-notebooks/edge/cpp-integration/README.md#install-dlr-on-windows